pkgs/development/python-modules/pynetbox/default.nix

```diff
@@ -0,0 +1,49 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, requests
+, six
+, pytestCheckHook
+, pyyaml
+}:
+
+buildPythonPackage rec {
+  pname = "pynetbox";
+  version = "6.1.2";
+
+  src = fetchFromGitHub {
+    owner = "netbox-community";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "0di07rny3gqdfb0rf7hm3x03rpn7rydpv3lrl7cak2ccpqm0wzhl";
+    leaveDotGit = true;
```

Comment:
Can we set SETUPTOOLS_SCM_PRETEND_VERSION instead?

Reply:
Neat! Thanks for the tip.